src/app/pages/FeedPage/components/Trend.tsx

```diff
@@ -26,14 +26,13 @@ const styles = {
     transition-colors duration-200
   `,
   container__name: `
-    flex 
-    text-black dark:text-white 
+    flex  
     font-bold 
     text-lg
   `,
   container__twits: `
     flex 
-    text-[#536471] dark:text-[#71767b] 
+    text-secondaryText-light dark:text-secondaryText-dark 
```

Comment:
See https://github.com/KirillKrm/twitter-clone/pull/2/commits/486877005f5645763d402c12b05a5a3c74c093a4#r1225380904

Reply:
Didn't understand. Where to look?

Reply:
Seems link was broken. That's the comment: https://github.com/KirillKrm/twitter-clone/pull/2#discussion_r1225371943